transit/transit_graph_data.cpp

```diff
@@ -184,8 +184,11 @@ void DeserializerFromJson::operator()(FeatureIdentifiers & id, char const * name
   string osmIdStr;
   GetField(osmIdStr, name);
   uint64_t osmIdNum;
-  CHECK(strings::to_uint64(osmIdStr, osmIdNum),
-        ("Cann't convert osm id string:", osmIdStr, "to a number."));
+  if (!strings::to_uint64(osmIdStr, osmIdNum))
```

Comment:
This code does exactly the same as what CHECK does but looks more complex. Are there other ways to properly fix the _wrong_ warning?

Reply:
> This code does exactly the same as what CHECK does but looks more complex. 

No, the warning comes in `debug` builds, where `CHECK` doesn't abort, and lets execution continue with potential memory garbage.

Reply:
`CHECK` always fails, either with std::abort in non-debug builds, or with assert(false) in debug ones. Can you please elaborate on the case when CHECK doesn't stop the execution in DEBUG? How did you reproduce it?